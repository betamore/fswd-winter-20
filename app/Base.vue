<template>
    <p>This is Base</p>
</template>
