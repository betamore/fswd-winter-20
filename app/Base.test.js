/**
 * @jest-environment jsdom
 */

import { shallowMount, mount } from "@vue/test-utils";
import Base from "./Base.vue";

describe("Base vue component", function() {
    it("should render something", function() {
        const wrapper = shallowMount(Base);
        expect(wrapper.text()).toBe("This is Base");
    });

    it("should make the snapshot", function() {
        const wrapper = mount(Base);
        expect(wrapper.element).toMatchSnapshot();
    });
});
