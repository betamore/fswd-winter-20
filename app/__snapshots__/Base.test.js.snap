// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base vue component should make the snapshot 1`] = `
<p>
  This is Base
</p>
`;
